src/views/Predictions/context/config.ts

```diff
@@ -0,0 +1,20 @@
+import addresses from 'config/constants/contracts'
+
+import { GRAPH_API_PREDICTION_CAKE, GRAPH_API_PREDICTION } from 'config/constants/endpoints'
+import { getAddress } from 'utils/addressHelpers'
+import tokens from 'config/constants/tokens'
+
+export default {
+  BNB: {
+    address: getAddress(addresses.predictions),
+    api: GRAPH_API_PREDICTION,
```

Comment:
name can have bnb suffix

Reply:
Good idea. I will do it after release because it requires a env config on Vercel.